<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChifoumiVue</class>
 <widget class="QMainWindow" name="ChifoumiVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChifoumiVue</string>
  </property>
  <widget class="QWidget" name="fenetrePrincipalH">
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="fenetreSecondaireV">
      <item>
       <layout class="QGridLayout" name="grilleTop">
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="hLayout_ChoixMachine">
          <item>
           <spacer name="hSpacerG_ChoixMachine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lChoixMachine">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ressourcesChifoumi.qrc">:/chifoumi/images/rien.gif</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacerD_ChoixMachine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="vLayout_JoueurScore">
          <item>
           <widget class="QLabel" name="label_Vous">
            <property name="text">
             <string>Vous</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lScoreJoueur">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lGagnant_a">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Gagnant Ã </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lScoreGagnant">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="hLayout_ChoixJoueur">
          <item>
           <spacer name="hSpacerG_ChoixJoueur">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lChoixJoueur">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ressourcesChifoumi.qrc">:/chifoumi/images/rien.gif</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacerD_ChoixJoueur">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="hLayout_Versus">
          <item>
           <layout class="QVBoxLayout" name="vLayout_Versus">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="hSpacerG_Versus">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="lVersus">
                <property name="minimumSize">
                 <size>
                  <width>78</width>
                  <height>78</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>78</width>
                  <height>78</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="ressourcesChifoumi.qrc">:/chifoumi/images/versus_70.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="hSpacerD_Versus">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hLayout_Versus_bottom">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="lTemps_restant">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string> Temps restant (s) : </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lSeconde">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>30</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="vLayout_MachineScore">
          <item>
           <widget class="QLabel" name="lMachine">
            <property name="text">
             <string>Machine</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lScoreMachine">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="vSpacerMiddle">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Choisissez une figure :</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="hSpacerG_GroupBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="boutonPierre">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="ressourcesChifoumi.qrc">
             <normaloff>:/chifoumi/images/pierre.gif</normaloff>:/chifoumi/images/pierre.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boutonPapier">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="ressourcesChifoumi.qrc">
             <normaloff>:/chifoumi/images/papier.gif</normaloff>:/chifoumi/images/papier.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boutonCiseau">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="ressourcesChifoumi.qrc">
             <normaloff>:/chifoumi/images/ciseau.gif</normaloff>:/chifoumi/images/ciseau.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpacerD_GroupBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="grilleBottom">
        <item row="1" column="0">
         <spacer name="hSpacerNvllePartie">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="boutonNvllePartie">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Nouvelle partie</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="boutonPause">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="action_a_propos_de"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_a_propos_de">
   <property name="text">
    <string>A propos de...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressourcesChifoumi.qrc"/>
 </resources>
 <connections/>
</ui>
